string milestone_name
---
bool goal_sent_status
